core/redfishCommand.py

```diff
@@ -124,12 +124,14 @@ def execute(cls, redfishConfig, command, echo = False):
         except ImportError as e:
             Trace.log(TraceLevel.ERROR, 'EXCEPTION: {}'.format(e))
             Trace.log(TraceLevel.ERROR, 'ImportError, No module found for command ({}) using [{}]'.format(command, handlerName))
-            pass
+            if Trace.getlevelint() < TraceLevel.DEBUG:
+                pass
```

Comment:
What purpose do these `pass` statements serve?  They look like `no-op`'s to me.  To my knowledge python 3 does not require them at the end of an `except` block if you do not `raise`.  Is this a style thing?

Reply:
The pass statements allow the code to catch the exception and continue running. Without them, the program terminates and prints a call stack. With the 'pass' statement in the trace messages are printed and execution continues.

Reply:
That has never been my experience with Python (2 or 3) and is contrary to the Python [docs](https://docs.python.org/3.6/tutorial/errors.html#handling-exceptions).  
See the end of bullet # 3:

-------------------------------------------------
The [try] statement works as follows.

- First, the try clause (the statement(s) between the [try] and [except] keywords) is executed.

- If no exception occurs, the except clause is skipped and execution of the [try] statement is finished.

- If an exception occurs during execution of the try clause, the rest of the clause is skipped. Then if its type matches the exception named after the [except] keyword, the except clause is executed, **_and then execution continues after the [try] statement_**.

- If an exception occurs which does not match the exception named in the except clause, it is passed on to outer [try] statements; if no handler is found, it is an unhandled exception and execution stops with a message as shown above.
-------------------------------------------------



I confirmed my expected behavior on one of our Linux servers using a simple example.
However, there may be some aspect of Python behavior here that I'm not yet familiar with.  If this is the case, then it's all good.  However, if not, I'd prefer to remove unnecessary statements.

Reply:
I also tested out what you stated and I agree with you that the 'pass' is not needed and I will remove them. I am not sure what has changed since initially adding them, but I prefer to remove them at this point. Thanks for the feedback.